app/views/settings/index.slim

```diff
@@ -86,62 +90,63 @@
-                            i Hidden forever
-                          - else
+                        th Name
+                        th Active Tokens
+                        th Redirect URI
+                        th Client ID
+                        th
+                          span> Client Secret &mdash;
+                          small.text-warning
+                            span = icon("fas", "clock")
+                            span< Hides forever after 24 hours
+                        th
+                    tbody
+                      - current_user.applications.each do |application|
+                        tr
+                          td
+                            big = application.name
+                          td = application.access_tokens.count
+                          td
+                            code = application.redirect_uri
+                          td
                             .input-group
-                              input.form-control id="client-secret-#{application.uid}" type='text' value=application.secret readonly=1
+                              input.form-control id="client-id-#{application.uid}" type='text' value=application.uid readonly=1
                               .input-group-btn
-                                .btn.btn-primary.clipboard-btn data-clipboard-target="#client-secret-#{application.uid}"
+                                .btn.btn-primary.clipboard-btn data-clipboard-target="#client-id-#{application.uid}"
                                   span#copy = icon("fas", "copy")
                                   span#copied style='display: none;' = icon("fas", "check")
-                        coffee:
-                          clipboard = new Clipboard('.clipboard-btn')
-                          clipboard.on('success', (e) ->
-                            e.trigger.children[0].style.display = 'none'
-                            e.trigger.children[1].style.display = 'inline-block'
-                            setTimeout(->
-                              e.trigger.children[0].style.display = 'inline-block'
-                              e.trigger.children[1].style.display = 'none'
-                            , 4000)
-                          )
-                        td
-                          small = button_to application_path(application), class: 'btn btn-primary', method: :delete, \
-                            data: { \
-                              confirm: "Really delete #{application.name}? This will also revoke all \
-                                        #{application.access_tokens.count} authorizations and invalidate the associated \
-                                        tokens." \
-                            }
-                            span = icon("fas", "times")
+                          td
+                            - if (Time.now - 24.hours) > application.created_at
+                              i Hidden forever
```

Comment:
This should probably be wrapped in a `btn.btn-primary` to maintain consistency with the client id.  Looks a bit out of place currently.

Reply:
The colors of a `.btn-primary` didn't look right for informational non-interactable text, but I went for a `.btn.btn-secondary.btn-block` with `disabled`:

<img width="776" alt="screenshot 2018-06-10 22 36 27" src="https://user-images.githubusercontent.com/438911/41214207-e82a60e6-6cfe-11e8-90ee-ff937451196b.png">